src/mpyl/cli/projects.py

```diff
@@ -98,8 +98,7 @@ def shell_complete(self, ctx: click.Context, param, incomplete: str):
         )
         found_projects = repo.find_projects(incomplete)
         return [
-            CompletionItem(value=proj.replace(f"/{Project.project_yaml_path()}", ""))
-            for proj in found_projects
+            CompletionItem(value=str(Path(proj).parents[1])) for proj in found_projects
```

Comment:
I see `.parents[1]` being used multiple times, maybe we should make another static function in `Project` to return the project root path from the full yaml path? That would make it a bit easier to maintain if the folder structure would ever change

Reply:
Addressed in https://github.com/Vandebron/mpyl/pull/287/commits/91bb7489d9cfdacc6f7855d6af138c8025478c99